templates/minio/statefulset.yaml

```diff
@@ -81,8 +97,18 @@ spec:
           successThreshold: 1
           failureThreshold: 3
         volumeMounts:
+        - mountPath: /tmp
+          name: liferay-minio-pvc
+          subPath: tmp-dir
+        - mountPath: /opt/bitnami/minio/tmp
+          name: liferay-minio-pvc
+          subPath: app-tmp-dir
+        - mountPath: /.mc
+          name: liferay-minio-pvc
+          subPath: app-mc-dir
         - mountPath: /bitnami/minio/data
           name: liferay-minio-pvc
+          subPath: data-dir
```

Comment:
the only problem adding `subPath` here might cause is that existing PVCs will have their existing data (which was previously in the root) in the wrong place. So I would suggest removing this and just making sure the other subdirs don't conflict with the original data (I think all the subPaths you have specified already are fine for this purpose)